### DISCUSSION TITLE
How can i setting different colors to differences between two images?

### DISCUSSION THREAD

**Discussion by coolwtu** [dlemstra/Magick.NET#904] [Help]
I want make 2 different colors on the output image when using Imagemagick.I have an original image with "Hello World" in it. And a modified image with "Hello Warcraft" in the same area. The default compare command will give me a image and mark all the differences with red.

Now I want to use 2 different colors like "orld" marked as red, and "arcraft" marked as another color, maybe blue. Is ImageMagick able to do this?

**Comment by dlemstra** [comment]
Can you add the images to this message so I can understand stand more clearly what you want. And this will also help me to play around with this.
